pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -103,6 +103,7 @@ func amendFilterChainMatchFromInboundListener(chain *listener.FilterChain, l *li
 }
 
 func isBindtoPort(l *listener.Listener) bool {
+	// nolint: staticcheck
```

Comment:
@howardjohn want to make sure it is ok to simply skip lint check.

Reply:
Yes I think this is fine, we are intentionally using this. And we have an issue around cleaning up deprecated field usage